tcf_website/templates/course/course_professor.html

```diff
@@ -194,7 +193,7 @@ <h2 class="mr-md-3">{{ course.title }}</h2>
 
         <div class="reviews-container mb-5 px-3">
             <div class="reviews-toolbar pb-2 d-flex flex-md-row flex-column justify-content-left justify-content-md-center">
-                <h3 class="reviews-heading mb-0 mr-4">{{ reviews|length }} Comments</h3>
+                <h3 class="reviews-heading mb-0 mt-1 mr-4">{{ reviews|length }} Comments</h3>
                 <div class="reviews-options d-flex flex-sm-row flex-column col p-0">
                     <a href="{% url 'new_review' %}" class="btn bg-tcf-orange text-white">
                       <i class="fa fa-plus" aria-hidden="true"></i> Add your review!
```

Comment:
This button still looks funky on mobile (maybe out of scope)
Probably adding some y margin would fix it
![image](https://user-images.githubusercontent.com/45245651/107159032-ae2c2080-695b-11eb-9ce7-3ea176df9e57.png)

Reply:
yeah lol oof i can't get them on the same line
I'll add some bottom margin!